### PR TITLE
skip email on completion page

### LINKED ISSUE #340: Payment completion page for anonymous payments
For anonymous payments (such as donations), we may not have an email address so the payment completion page needs to be a little different.

Supports: https://github.com/mozilla/payments/issues/117

### LINKED ISSUE #348: Fix up other uses of per month
In the receipts and in the management interface.

They should be checking the product config and probably make a little utility function out of that.

### PR DESCRIPTION
**Pull request by andymckay**
* fixes #340 
* I feel like doing these if's might not scale, but also breaking everything down into smaller and smaller parts seem a little crazy
* seems to work, but guidance welcome

### CODE REVIEW

File: public/js/views/transaction/complete-payment.jsx [hunk 1]
```diff
@@ -42,16 +42,19 @@ export default class CompletePayment extends Component {
...
+      </p> : '');
+
     return (
       <div className="complete">
         <ProductDetail productId={this.props.productId} />
         <p className="accepted">{gettext('Payment Accepted')}</p>
-        <p className="receipt">
-          {gettext('Your receipt has been sent to')}
-          <span className="email">{this.props.userEmail}</span>
-        </p>
```
Comment (kumar303) [inline public/js/views/transaction/complete-payment.jsx]: I think `{email}` should go here, not below the submit button to follow the mocks

**Comment by kumar303** [comment]
> I feel like doing these if's might not scale

I actually like this approach because it's very direct: it only checks for the presence of email which is the right kind of if statement

File: public/js/views/transaction/complete-payment.jsx (hunk 1 above)
Comment (andymckay) [inline public/js/views/transaction/complete-payment.jsx]: oh oops, thanks

**Comment by kumar303** [comment]
r+

**Pull request merged**